lib/debug/observatory_debugger.dart

```diff
@@ -496,7 +505,15 @@ class ObservatoryIsolate extends DebugIsolate {
   stepIn() => service.resume(isolateRef.id, step: StepOption.kInto);
   stepOver() => service.resume(isolateRef.id, step: StepOption.kOver);
   stepOut() => service.resume(isolateRef.id, step: StepOption.kOut);
-
+  stepOverAsyncSuspension() =>
+      service.resume(isolateRef.id, step: StepOption.kOverAsyncSuspension);
```

Comment:
Ok, so this will only send `kOverAsyncSuspension` into a newer client, as older clients will cause the `atAsyncSuspension` field to populate w/ false above.